app/models/agents/pushover_agent.rb

```diff
@@ -95,6 +97,16 @@ def receive(incoming_events)
               post_params[key] = value
             end
           end
+          # html is special because String.try_convert(true) gives nil (not even "nil", just nil)
+          if value = interpolated['html'].presence
+            post_params['html'] =
+              case value.to_s
+              when 'true', '1'
```

Comment:
Is there a reason why we want to support `1` as true here? We normally rely on the [`boolify`](https://github.com/huginn/huginn/blob/fdf4b0794aaed8a342d0dbc1658b43f52814982b/app/models/agent.rb#L281) that only accepts `true` and `"true"`.

Reply:
Pushover's API documentation gives `1` as the value to pass, so it made sense to support what their docs said to do.